### PR TITLE
support for python 3

### PR DESCRIPTION
**Pull request by ragupta-git**
changes made in auto-generated corefiles to make ucsmsdk work with python3 too. In parallel meta data too changed.

modifed methodmeta and mometa to make ucsmsdk work with python3 too.

modifed ccoimage to make it work with python3 too.

modifed converttopython to make it work with python3 too.

modifed ucsguilaunch to make it work with python3 too.

modifed ucskvmlaunch to make it work with python3 too.

modified ucsxmlcodec to work with python3 too

few more changes to support python version 3 too

modified converttopython to support python version3 too

modified tests for python3 version support and added license under sub tests

### CODE REVIEW

File: tests/common/test_request_xml.py [hunk 1]
```diff
@@ -37,7 +39,7 @@ def commit():
     refresh_dict = {}
     mo_dict = to_commit
     if not mo_dict:
-        print "No Mo to be Committed"
+        print("No Mo to be Committed")
```
Comment (vvb) [inline tests/common/test_request_xml.py]: May be we should extract print out to a utils file. And handle Python 2 vs Python 3 logic there. Applies to all other print statements too.

File: ucsmsdk/methodmeta/AaaLoginMeta.py [hunk 2]
```diff
@@ -1,9 +1,7 @@
 """This module contains the meta information of AaaLogin ExternalMethod."""
 import sys, os
 
-sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
-from ucscoremeta import MethodMeta, MethodPropertyMeta
-sys.path.remove(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+from ..ucscoremeta import MethodMeta, MethodPropertyMeta
```
Comment (vvb) [inline ucsmsdk/methodmeta/AaaLoginMeta.py]: Add a note in Readme about the minimum Python version required to support this.

File: ucsmsdk/mometa/aaa/AaaAuthRealm.py [hunk 3]
```diff
@@ -195,12 +193,12 @@ class AaaAuthRealm(ManagedObject):
...
-        "policy_owner": MoPropertyMeta("policy_owner", "policyOwner", "string", VersionMeta.Version211a, MoPropertyMeta.READ_WRITE, 0x100L, None, None, None, ["local", "pending-policy", "policy"], []), 
-        "rn": MoPropertyMeta("rn", "rn", "string", VersionMeta.Version101e, MoPropertyMeta.READ_ONLY, 0x200L, 0, 256, None, [], []), 
-        "sacl": MoPropertyMeta("sacl", "sacl", "string", VersionMeta.Version302c, MoPropertyMeta.READ_ONLY, None, None, None, r"""((none|del|mod|addchild|cascade),){0,4}(none|del|mod|addchild|cascade){0,1}""", [], []), 
-        "status": MoPropertyMeta("status", "status", "string", VersionMeta.Version101e, MoPropertyMeta.READ_WRITE, 0x400L, None, None, r"""((removed|created|modified|deleted),){0,3}(removed|created|modified|deleted){0,1}""", [], []), 
+        "policy_owner": MoPropertyMeta("policy_owner", "policyOwner", "string", VersionMeta.Version211a, MoPropertyMeta.READ_WRITE, 0x100, None, None, None, ["local", "pending-policy", "policy"], []), 
+        "rn": MoPropertyMeta("rn", "rn", "string", VersionMeta.Version101e, MoPropertyMeta.READ_ONLY, 0x200, 0, 256, None, [], []),
```
Comment (vvb) [inline ucsmsdk/mometa/aaa/AaaAuthRealm.py]: Need to check why did the minversion for sacl change

**Comment by vvb** [comment]
Thanks Rahul. LGTM other than the minor comments. This brings the much needed Python 3 support.

File: ucsmsdk/mometa/aaa/AaaAuthRealm.py (hunk 3 above)
Comment (ragupta-git) [inline ucsmsdk/mometa/aaa/AaaAuthRealm.py]: Seems below version is missed in ucsmsdk version 0.9.0.0 processing.Hence the minimum version now changed to older one.
Version224a = UcsVersion("2.2(4a)")
Version302a = UcsVersion("3.0(2a)")

Comment (vvb) [inline ucsmsdk/mometa/aaa/AaaAuthRealm.py]: for `sacl`, I see that it changed from 3.0.2f to 3.0.2a. Not sure I understand what 2.2.4a has to do with it.

**Pull request merged**